### ISSUE TITLE
could not find recipe mod_php5 for cookbook apache2

### ISSUE THREAD

**Issue by ghost** [sous-chefs/nagios#546]
### Cookbook version
7.2.7

### Chef-client version
13.2.20

### Platform Details
CentOS Linux release 7.3.1611 (Core)

### Scenario:
Install nagios

## Apache2 chef supermarket 5.0.0
---------------------------------------
Note the apache2 recipe mod_php5 is now named mod_php
```
~/Cookbooks/apache2-5.0.0/recipes:$ ls mod_php*.rb
mod_php.rb
```

```
---------------------------------------------------------------------------
[2017-08-02T10:22:47-04:00] INFO: Setting up Nagios server via Apache2
==> default:
==> default: ================================================================================
==> default: Recipe Compile Error
==> default: ================================================================================
==> default:
==> default: Chef::Exceptions::RecipeNotFound
==> default: --------------------------------
==> default: could not find recipe mod_php5 for cookbook apache2
==> default:
==> default: Cookbook Trace:
==> default: ---------------
==> default:   /var/chef/cache/cookbooks/nagios/recipes/apache.rb:21:in `from_file'
==> default:   /var/chef/cache/cookbooks/nagios/recipes/default.rb:45:in `from_file'
==> default:   /var/chef/cache/cookbooks/acme-nagios-server/recipes/default.rb:26:in `from_file'
==> default:
==> default: Relevant File Content:
==> default: ----------------------
==> default: /var/chef/cache/cookbooks/nagios/recipes/apache.rb:
==> default:
==> default:  14:  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
==> default:  15:  # See the License for the specific language governing permissions and
==> default:  16:  # limitations under the License.
==> default:  17:  #
==> default:  18:
==> default:  19:  include_recipe 'apache2'
==> default:  20:  include_recipe 'apache2::mod_rewrite'
==> default:  21>> include_recipe 'apache2::mod_php5'
==> default:  22:  include_recipe 'apache2::mod_ssl' if node['nagios']['enable_ssl']
==> default:  23:
==> default:  24:  apache_site '000-default' do
==> default:  25:    enable false
==> default:  26:  end
==> default:  27:
==> default:  28:  apache_module 'cgi'
==> default:  29:
==> default:  30:  template "#{node['apache']['dir']}/sites-available/#{node['nagios']['server']['vname']}.conf" do
==> default:
==> default: System Info:
==> default: ------------
==> default: chef_version=13.2.20
==> default: platform=centos
==> default: platform_version=7.3.1611
==> default: ruby=ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
==> default: program_name=chef-client worker: ppid=8050;start=10:22:37;
==> default: executable=/opt/chef/bin/chef-client
```

**Comment by mtory** [comment]
mod_php5 is deprecated. Use mod_php instead.
https://github.com/sous-chefs/apache2/commit/2c030dc8ec0daed5f02d72793b315be6f966ffe8

**Comment by shoekstra** [comment]
This has been resolved in the v8.0.0 release which should land in the supermarket in the next couple of hours.  [Watch this space](https://github.com/sous-chefs/nagios/pull/551).

**Comment by ghost** [comment]
It's fixed.  Thanks Stephen.
